docs/cli.md

```
@@ -539,6 +539,19 @@ toolkit's `/core` and `/reporter` entry points are loaded via dynamic
 `import()` only inside the `analyze` code path, so every other CLI mode is
 unaffected if the dependency is ever missing.
 
+## export-k6
+
+Export a scenario JSON as a runnable [k6](https://k6.io) load-test bundle:
+
+    ocpp-cp-sim export-k6 --scenario charge.json -o loadtest/
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)